lms/envs/production.py

```diff
@@ -1137,6 +1137,31 @@
     COLARAZ_PREVIEW_DOMAIN_KEY
 )
 
+# Custom urls in navbar
+COLARAZ_RECRUITMENT_URL = ENV_TOKENS.get(
+    'COLARAZ_RECRUITMENT_URL'
+)
+COLARAZ_MANAGEMENT_URL = ENV_TOKENS.get(
+    'COLARAZ_MANAGEMENT_URL'
+)
+COLARAZ_PERFORMANCE_URL = ENV_TOKENS.get(
+    'COLARAZ_PERFORMANCE_URL'
+)
+COLARAZ_DEVELOPMENT_URL = ENV_TOKENS.get(
+    'COLARAZ_DEVELOPMENT_URL'
+)
+COLARAZ_SETTINGS_URL = ENV_TOKENS.get(
+    'COLARAZ_SETTINGS_URL'
+)
+COLARAZ_RECOMMENDATIONS_URL = ENV_TOKENS.get(
+    'COLARAZ_RECOMMENDATIONS_URL'
+)
+COLARAZ_MENTORING_URL = ENV_TOKENS.get(
+    'COLARAZ_MENTORING_URL'
+)
+COLARAZ_SCHOLARSHIP_URL = ENV_TOKENS.get(
+    'COLARAZ_SCHOLARSHIP_URL'
+)
```

Comment:
Instead of having these individual URL variables, what if we made a dict having some kind of URL map?
Just like edX does with the marketing URLs ?

Reply:
done